### PR TITLE
docs: second review on components 

### PR DESCRIPTION
**Pull request by carollalves**
## Purpose

_Second review on components_

### CODE REVIEW

File: docs/content/components/button.mdx [hunk 1]
```diff
@@ -136,6 +136,8 @@ with a primary button for the expected action and a secondary button on its left
 
 <Props of={ButtonGroup} />
 
+- **Do** use the same verb tenses for both actions
```
Comment (mlent) [inline docs/content/components/button.mdx]: WDYT about adding a subheadline like `### Content guidelines` on top of this? :)

**Comment by connor-baer** [comment]
What's the status of this PR? 

We will start work on migrating the docs from Docz to Storybook Docs soon (#463) and I would like to merge these changes beforehand.

**Pull request merged**